webots_ros2_tutorials/protos/Robot_sense.proto

```diff
@@ -0,0 +1,203 @@
+PROTO Robot_sense [
```

Comment:
```suggestion
PROTO RobotSense [
```
Check the naming convention here:
https://www.cyberbotics.com/doc/reference/proto-design-guidelines#naming
(don't forget to change the name in `setup.py` and the filename)

The file is also missing a header:
https://www.cyberbotics.com/doc/reference/proto-design-guidelines?version=develop#example